### PR TITLE
Support authorization

### PR DESCRIPTION
**Pull request by oeway**

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Maybe rebase on latest master branch to avoid the old commits in the branch?

**Comment by oeway** [comment]
I tried to rebase, but it's too complicated to resolve the conflicts, so I give up and merge master instead. The file changes looks good now. would this be a big issue later?

**Comment by MartinHjelmare** [comment]
I'd and interactive rebase and squash the new commits that you added to the old branch, and drop all the old commits before that.

**Comment by MartinHjelmare** [comment]
The squash may not be needed, but it may simplify resolving potential merge conflicts.

**Comment by MartinHjelmare** [comment]
I've rebased the branch locally. Let me know if I can push.

**Comment by oeway** [comment]
> I've rebased the branch locally. Let me know if I can push.


I tried it again and messed up with the rebase.


That's great,  could you get 00e3346 to your local branch? then please go ahead and push. I will wait for your rebase.

**Comment by MartinHjelmare** [comment]
Done! :+1:

**Comment by oeway** [comment]
> Done! 👍

Thank you!

File: imjoy/VERSION [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "version": "0.11.0"
+    "version": "0.11.1"
```
Comment (MartinHjelmare) [inline imjoy/VERSION:2]: I'd bump minor version since this is a new feature.

**Pull request merged**